pkg/translate/output.go

```diff
@@ -79,16 +80,24 @@ func printDisconnectedBlocks(wr io.Writer, root *common.Vertex, domainVisited ma
 	return nil
 }
 
-func verifyBlockSize(domainVisited map[string]int) int {
-	blockSize := -1
-	for _, bSize := range domainVisited {
-		if blockSize == -1 {
-			blockSize = bSize
-		} else if blockSize != bSize {
-			fmt.Printf("Alert! blockSize different between NVL domains\n")
+func getBlockSize(domainVisited map[string]int, adminBlockSize string) string {
+	minDomainSize := -1
+	for _, dSize := range domainVisited {
+		if minDomainSize == -1 || minDomainSize > dSize {
+			minDomainSize = dSize
 		}
 	}
-	return blockSize
+	if adminBlockSize != "" {
+		blockSizes := strings.Split(adminBlockSize, ",")
+		planningBS, _ := strconv.Atoi(blockSizes[0])
```

Comment:
I would add checking error here, in case the input is not valid, in which case we can ignore provided block size

Reply:
Added.